Core/FileLoaders/LocalFileLoader.cpp

```diff
@@ -15,23 +15,25 @@
 // Official git repository and contact information can be found at
 // https://github.com/hrydgard/ppsspp and http://www.ppsspp.org/.
 
-#include <cstdio>
 #include "file/file_util.h"
 #include "Common/FileUtil.h"
 #include "Core/FileLoaders/LocalFileLoader.h"
+#ifdef _WIN32
+#include "Common/CommonWindows.h"
+#endif
 
 LocalFileLoader::LocalFileLoader(const std::string &filename)
 	: fd_(0), f_(nullptr), filesize_(0), filename_(filename) {
+    // FIXME: perhaps at this point we should just use plain open?
```

Comment:
Windows could use [OpenFileW](https://msdn.microsoft.com/en-us/library/windows/desktop/aa365430(v=vs.85).aspx), of course. Which would also avoid having to use internal-ish API to convert the FD into HANDLE.